src/routes/safes/safes.controller.ts

```diff
@@ -10,6 +10,7 @@ import { SafesService } from '@/routes/safes/safes.service';
 export class SafesController {
   constructor(private readonly service: SafesService) {}
 
+  @ApiOkResponse({ type: SafeState })
```

Comment:
I might have missed this 😬 

I added it in https://github.com/safe-global/safe-client-gateway/commit/1ada6bdd298179987f4fb0b8dfb65a809d6a6be5